ndcube/tests/test_ndcube.py

```diff
@@ -309,7 +309,7 @@ def test_slicing_second_axis(test_input, expected, mask, wcs, uncertainty,
 
 @pytest.mark.parametrize(
     "test_input,expected,mask,wcs,uncertainty,dimensions,world_axis_physical_types, extra_coords",
-    [(cubem[1],  #13
+    [(cubem[1],
       NDCube,
       mask_cubem[1],
       _wcs_slicer(wm, [False, False, False], 1),
```

Comment:
Do we need the ```_wcs_slicer``` call split over three lines?

Reply:
It's been like this for a while. Would you like me to change it?